x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/top_values/_top_values.scss

```diff
@@ -0,0 +1,11 @@
+.mlTopValuesLabelContainer {
+  padding-right: $euiSizeM;
+
+  &.small {
+    width: 50px !important;
+  }
+
+  &.large {
+    width: 200px !important;
+  }
+}
```

Comment:
We should use a different structure, with this we end up with non-prefixed very generic css classes in the DOM like `small/large`. Consider using the BEM-style described on the bottom of this page (Sass best practices) https://elastic.github.io/eui/#/guidelines/sass.

You should end up creating classes that look like `mlTopValuesLabelContainer--small`. In the react code you will need to use both the base class and this modifier.

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/0dcdf2efbdbfdbd6a0259f38cc37bac7942cfae7